### ISSUE TITLE
"The provided training data is very small for code verification purpose only"

### ISSUE THREAD

**Issue by sbkim052** [auspicious3000/SpeechSplit#18]
Is this mean the pre-trained model given is kind of overfitted model trained on small dataset?

**Comment by auspicious3000** [comment]
no

**Comment by sbkim052** [comment]
Then is it possible to convert the voices of our own data with your pre-trained model? will it perform well?

**Comment by auspicious3000** [comment]
no, the pretrained model only works for speakers in the training set

**Comment by FurkanGozukara** [comment]
> no, the pretrained model only works for speakers in the training set

How can I test my own speech data with your pre-trained model?

Lets say how can I train a model for the voice of trump and apply it to my speech?

**Comment by FurkanGozukara** [comment]
@sbkim052  can you check my problem? did you pass these steps?

https://github.com/auspicious3000/SpeechSplit/issues/28

**Comment by leijue222** [comment]
> no, the pretrained model only works for speakers in the training set

Can I use it to train a model like Conan's Bow? 
Anyone's voice will be converted to the model's voice.

**Comment by leijue222** [comment]
Or this can only use the data in the training set, and can't convert the voice of a new speaker into the pre-trained model's voice.

**Comment by auspicious3000** [comment]
@leijue222 yes you can, but you need to re-train the model.

**Comment by leijue222** [comment]
> @leijue222 yes you can, but you need to re-train the model.

That's wonderful.
Actually, I want to use the [baker's](https://www.data-baker.com/#/data/index/source) voice as Conan's bow-knot, with your SpeechSplit to do voice conversion.
Then, the SpeechSplit can convert my voice or your voice or anyone else voice to baker's voice.
